### ISSUE TITLE
The Openflexure Adaptor misses images and needs an update

### ISSUE THREAD

**Issue by beniroquai** [bionanoimaging/UC2-GIT#75]
I just wanted to add this so that we won't forget. The [page](https://github.com/bionanoimaging/UC2-GIT/tree/master/CAD/CUBE_INSERT_OpenFlexure) misses images. Perhaps we can also update the parts? Are they up-to-date?

**Comment by b286** [comment]
Images restored with latest commit, they got lost on the way. I don't think it's up-to-date, but I've never worked with it myself. Maybe @ranranking uses the latest version?

**Comment by ranranking** [comment]
You mean the single adapter for Openflexure? I will try to find it and post image here, you can check if is that one

**Comment by beniroquai** [comment]
That would be great. Then we can close this thing again :)

>

**Comment by Tomvhattem** [comment]
I see there is only an adapter for the bottom. We also made an adapter to fit on the top, is that something we can add to this repo?

**Comment by beniroquai** [comment]
Certainly! You can fork it and then file a pull request! That would be wonderful :)
